vijaya/Boostrap/assignment3/assignment3.html

```diff
@@ -0,0 +1,117 @@
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" />
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+        <link rel="stylesheet" href="assign1.css" />
+        <title>Boostrap Assignment 3</title>
+        <style>
+            .heading{
+                color: rgb(67, 67, 156);
+                font-weight: bold;
+            }
+            .heading_color{
+                color: rgb(189, 30, 189);
+                font-weight: bold;
+            }
+            .heading_color1{
+                color: rgb(34, 36, 150);
+                font-weight: bold;
+            }
+            .green{
+                width:150px;
+                height: 152px;
+                background-color:#fff;
+                border: 2px solid #000;
+                border-radius: 50%;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+            }
+            .green2{
+                width:150px;
+                height: 152px;
+                background-color:#fff;
+                border: 2px solid #000;
+                border-radius: 50%;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                margin-left: 68px;
+            }
+            .circle {
+                position: relative;
+                width: 138px;
+                height: 144px;
+                border: 1px solid #fff;
+                border-radius: 50%;
+                margin-top: 16px;
+                margin-right: 11px;
+                margin-left: -65px;
+            }
+            .circle2 {
+                position: relative;
+                width: 138px;
+                height: 144px;
+                border: 1px solid #fff;
+                border-radius: 50%;
+                margin-top: 10px;
+                margin-right: -67px;
+                margin-left: -66px;
+            }
+            .circle::after {
+                content: '';
+                width: 150px;
+                height: 150px;
+                border: 1px solid #fff;
+                border-radius: 50%;
+                display: block;
+                margin: -4px 2px;
+            }
+        </style>
+    </head>
+    <body>
+        <h1 class="text-center"><span class="heading">About</span> Us</h1>
+        <hr>
+        <hr>
+        <p style="text-align: center;">Downloading this awesome diagram.Bring your presentation to life.Capture your audience attention.</p><br>
+        <div class="container">
+             <div class="row">
```

Comment:
check indentation extra space in this line

Reply:
ok done